scene/resources/packed_scene.cpp

```diff
@@ -123,19 +123,111 @@ Ref<Resource> SceneState::get_remap_resource(const Ref<Resource> &p_resource, Ha
 	return remap_resource;
 }
 
+NodePath SceneState::adjust_if_replaced_path(NodePath p_path, const NodePath *p_node_source_path, const List<NodePathRemapData> &p_remapped_paths) const {
+	String path_string = p_path;
+	String based_path;
+	if (p_node_source_path != nullptr && path_string.begins_with(".")) { // Should be relative
+		based_path = NodePath(String(*p_node_source_path).path_join(path_string)).simplified();
+	}
+	for (const NodePathRemapData &rd : p_remapped_paths) {
+		if (!based_path.is_empty()) {
+			if (based_path.contains(rd.original_path)) {
+				based_path = based_path.replace(rd.original_path, rd.adjusted_path);
+				NodePath return_value = NodePath("/" + String(*p_node_source_path)).rel_path_to("/" + based_path);
+				print_verbose(vformat("Adjusting changed relative NodePath '%s' to '%s'", p_path, return_value));
+				return return_value;
+			}
+		} else {
+			if (path_string.contains(rd.original_path)) {
+				path_string = path_string.replace(rd.original_path, rd.adjusted_path);
+				print_verbose(vformat("Adjusting changed absolute NodePath '%s' to '%s'", p_path, path_string));
+				return NodePath(path_string);
+			}
+		}
+	}
+	return p_path;
+}
+
+bool SceneState::adjust_if_replaced_path_recursive(const StringName *p_property_name, const NodePath *p_node_source_path, const List<NodePathRemapData> &p_remapped_paths, Variant &value, int p_current_depth) const {
+	ERR_FAIL_COND_V_MSG(p_current_depth > 5, value, vformat("Property depth limit of 5 exceeded in '%s', further NodePath adjustments will be skipped", p_property_name));
```

Comment:
```suggestion
	ERR_FAIL_COND_V_MSG(p_current_depth > 5, value, vformat("Property depth limit of 5 exceeded in '%s', further NodePath adjustments will be skipped", *p_property_name));
```
The argument is a pointer 🙂

Reply:
Another oversight, thx